CrazyCanvas/Source/GUI/KillFeedGUI.cpp

```diff
@@ -44,28 +45,68 @@ void KillFeedGUI::UpdateFeedTimer(LambdaEngine::Timestamp delta)
 	}
 }
 
-void KillFeedGUI::AddToKillFeed(const LambdaEngine::String& feedMessage, const uint8 killedPlayerTeamIndex)
+void KillFeedGUI::AddToKillFeed(const LambdaEngine::String& killed, const LambdaEngine::String& killer, const uint8 killedPlayerTeamIndex)
 {
-	Noesis::Ptr<Noesis::TextBlock> feed = *new Noesis::TextBlock();
-	Noesis::Ptr<Noesis::SolidColorBrush> pBrush = *new Noesis::SolidColorBrush();
-	
-	uint8 colorIndex = killedPlayerTeamIndex == 0 ? 1 : 0; //will only work for 2 teams, Implement a smarter way to handle this later
-	glm::vec3 teamColor = TeamHelper::GetTeamColor(colorIndex);
-	Noesis::Color killFeedColor(teamColor.r, teamColor.g, teamColor.b);
+	Noesis::Ptr<Noesis::Grid> grid = *new Noesis::Grid();
 
-	pBrush->SetColor(killFeedColor);
+	AddColumnDefinition(grid->GetColumnDefinitions(), 1.0f, Noesis::GridUnitType::GridUnitType_Star);
+	AddColumnDefinition(grid->GetColumnDefinitions(), 1.0f, Noesis::GridUnitType::GridUnitType_Star);
+	AddColumnDefinition(grid->GetColumnDefinitions(), 1.0f, Noesis::GridUnitType::GridUnitType_Star);
 
-	feed->SetText(feedMessage.c_str());
-	feed->SetFontSize(23.0f);
-	feed->SetForeground(pBrush);
+	Noesis::Ptr<Noesis::TextBlock> killerFeed	= *new Noesis::TextBlock();
+	Noesis::Ptr<Noesis::TextBlock> feed			= *new Noesis::TextBlock();
+	Noesis::Ptr<Noesis::TextBlock> killedFeed	= *new Noesis::TextBlock();
 
```

Comment:
Looks like you are inconsistent with the naming convension. SmartPtrs can be called with or without prefix p. I think you should stick to one way.

Reply:
Can you please show me where I'm naming something declared with Noesis::Ptr<xxx> with a prefix p? and something not declared with Noesis::Ptr<xxx> without a p as prefix?